test/conversionRate.js

```diff
@@ -87,7 +87,6 @@ contract('ConversionRates', function(accounts) {
     it("should init ConversionRates Inst and set general parameters.", async function () {
         //init contracts
         convRatesInst = await ConversionRates.new(admin);
-
```

Comment:
please avoid these kind of changes.
try to eliminate as much changes as possible...